test/gtest/group_conv3d_bwd.cpp

```diff
@@ -54,8 +54,9 @@ void SolverBwd(const miopen::TensorDescriptor& inputDesc,
     const auto tensors =
         miopen::ConvBwdTensors{outputDesc, output, wDesc, weight, inputDesc, input};
 
+    // order for bwd data pass is dy, w, dx
     const auto problem = miopen::conv::ProblemDescription{
-        inputDesc, wDesc, outputDesc, convDesc, miopen::conv::Direction::BackwardData};
+        outputDesc, wDesc, inputDesc, convDesc, miopen::conv::Direction::BackwardData};
```

Comment:
[Notice] The test (incorrectly) uses notions of "input" and "output", which is not good for convolutions. This is legacy that we should get rid of in the future, I guess.

Reply:
Lets make that the "near future"